tests/jquery.flot.hover.Test.js

```diff
@@ -258,5 +258,81 @@ describe("flot hover plugin", function () {
 
             expect(getEntireCanvasData(canvas)).toContainPixelColor(rgba(10, 20, 30, 1));
         });
+
+        it('should call hooked function when canvas is hovered over', function() {
+            var spy = jasmine.createSpy('spy');
+
+            options.hooks = {findNearbyItems: [spy]};
+            plot = $.plot(placeholder, [ [ [0, 0], [2, 3], [10, 10] ] ], options);
+
+            var eventHolder = plot.getEventHolder(),
+                offset = plot.getPlotOffset(),
+                axisx = plot.getXAxes()[0],
+                axisy = plot.getYAxes()[0],
+                x = axisx.p2c(2) + offset.left,
+                y = axisy.p2c(3) + offset.top,
+                noButton = 0;
+
+            simulate.mouseMove(eventHolder, x, y, noButton);
+            jasmine.clock().tick(100);
```

Comment:
We should remove the `jasmine.clock().tick()` calls from these tests if possible, and use callbacks and `done()` instead.

Reply:
I think this is a larger, general tech debt effort. We should capture that work separately.